docs/en/observability/apm/getting-started-apm-server/index.asciidoc

```diff
@@ -0,0 +1,153 @@
+[[apm-getting-started-apm-server]]
+= Get started with APM Server
+
+++++
+<titleabbrev>Get started</titleabbrev>
+++++
+
+TIP: The easiest way to get started with Elastic APM is by using our
+{ess-product}[hosted {es} Service] on {ecloud}.
+The {es} Service is available on AWS, GCP, and Azure.
+See <<get-started-with-fleet-apm-server,getting started documentation>> to get started in minutes.
+
+
+// TODO: MOVE THIS
+IMPORTANT: Starting in version 8.0.0, {fleet} uses the APM integration to set up and manage APM index templates,
+{ilm-init} policies, and ingest pipelines. APM Server will only send data to {es} _after_ the APM integration has been installed.
+
+The APM Server receives performance data from your APM agents,
+validates and processes it, and then transforms the data into {es} documents.
+If you're on this page, then you've chosen to self-manage the Elastic Stack,
+and you now must decide how to run and configure the APM Server.
+There are two options, and the components required are different for each:
+
+* **<<apm-setup-apm-server-binary,APM Server binary>>**
+* **<<apm-setup-fleet-managed-apm,Fleet-managed APM Server>>**
+// * **<<setup-apm-server-ea,Standalone Elastic Agent-managed APM Server>>**
+
+[float]
+[[apm-setup-apm-server-binary]]
+== APM Server binary
+
+Install, configure, and run the APM Server binary wherever you need it.
+
+image::./images/bin-ov.png[APM Server binary overview]
+
+[cols="1,3"]
+|===
+| *Pros*
+a| * Simplest self-managed option
```

Comment:
At the top of the page we link to the fleet server method and say that's the easiest way to get started. Here we say the opposite.

Reply:
Sorry, was not implying you did this. I think we should revisit. Guidance has likely changed since this page was created and this is confusing.

Reply:
Tracking in https://github.com/elastic/observability-docs/issues/4336